app/controllers/profiles_controller.rb

```diff
@@ -1,5 +1,7 @@
 class ProfilesController < ApplicationController
   before_action :redirect_to_signin, unless: :signed_in?, except: :show
+  before_action :redirect_to_new_mfa, if: :mfa_required_not_yet_enabled?
+  before_action :redirect_to_settings_strong_mfa_required, if: :mfa_required_weak_level_enabled?
```

Comment:
any reason why we aren't adding `except: :show` to these?

Reply:
There weren't any exceptions made in https://github.com/Shopify/rubygems.org/pull/10/files so I wasn't really sure for this one and the below comment.  I guess mirroring what the sign in checks do would be fine?